app.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+let express = require('express');
+let path = require('path');
+let favicon = require('serve-favicon');
```

Comment:
I just used express-generator to scaffold it. It's probably a lot easier to read / understand for beginners without all that stuff lying about, so removing them might be good. I also don't mind either way.